File: second_semester/POO/Pizarra/prueba.js
//creacion del metodo
class Zapato{
    constructor(talla){
        this.talla = talla;
    }
   //definimos el metodo
    mostrar_talla(){
        return `El zapato es número ${this.talla}`;

    }
}
//creacion del objeto
const zapato1 = new Zapato(25);
//ejecucion del metodo
console.log(zapato1.mostrar_talla());

File: second_semester/POO/Pizarra/prueba.py
#definimos la clase
class Zapato:
    def __init__(self,talla):
        self.talla = talla
#definimos metodo    
    def mostrar_talla(self):
        return f'El zapato es número {self.talla}'
#creación de objeto
zapato1 = Zapato(25)
#ejecución del metodo
print(zapato1.mostrar_talla())